epub33/rs/index.html

```diff
@@ -1835,7 +1835,10 @@ <h5>Text-to-Speech</h5>
 						associated speech synthesis. The implicit duration of the <code>text</code> element (and by
 						inference, of the parent <code>par</code> element) is therefore determined by the execution of
 						the Text-to-Speech engine, and cannot be known at authoring time (factors like speech rate,
-						pauses and other prosody parameters influence the audio output).</p>
+						pauses and other prosody parameters influence the audio output).
+						This also means that the <a data-cite="epub-33#duration"><code>duration</code></a>
+							property values set in the package document at authoring time may be considered as approximative only, and should be considered as such by the Reading System.
```

Comment:
Isn't their being approximative more definitive than the current statement suggests? I'd simplify this more along the lines of:

> This also means that Reading Systems should treat the `duration` property values set in the Package Document as approximative only when making use of them.

(Seems almost normative, except we don't define how to make use of them.)

Reply:
Adopted the change